connectors/README.md

```diff
@@ -1,24 +1,4 @@
-# <img src="https://docs.delta.io/latest/_static/delta-lake-white.png" width="150" alt="Delta Lake Logo"></img> Connectors
-
-[![Test](https://github.com/delta-io/connectors/actions/workflows/test.yaml/badge.svg)](https://github.com/delta-io/connectors/actions/workflows/test.yaml)
-[![License](https://img.shields.io/badge/license-Apache%202-brightgreen.svg)](https://github.com/delta-io/connectors/blob/master/LICENSE.txt)
-
-We are building connectors to bring [Delta Lake](https://delta.io) to popular big-data engines outside [Apache Spark](https://spark.apache.org) (e.g., [Apache Hive](https://hive.apache.org/), [Presto](https://prestodb.io/), [Apache Flink](https://flink.apache.org/)) and also to common reporting tools like [Microsoft Power BI](https://powerbi.microsoft.com/).
-
-# Introduction
-
-This is the repository for Delta Lake Connectors. It includes
-- [Delta Standalone](https://docs.delta.io/latest/delta-standalone.html): a native library for reading and writing Delta Lake metadata.
-- Connectors to popular big-data engines (e.g., [Apache Hive](https://hive.apache.org/), [Presto](https://prestodb.io/), [Apache Flink](https://flink.apache.org/)) and to common reporting tools like [Microsoft Power BI](https://powerbi.microsoft.com/).
-
-Please refer to the main [Delta Lake](https://github.com/delta-io/delta) repository if you want to learn more about the Delta Lake project.
-
-# API documentation
-
-- Delta Standalone [Java API docs](https://delta-io.github.io/connectors/latest/delta-standalone/api/java/index.html)
-- Flink/Delta Connector [Java API docs](https://delta-io.github.io/connectors/latest/delta-flink/api/java/index.html)
-
-# Delta Standalone
+## Delta Standalone
```

Comment:
This intro is not needed. 
Ideally we should merge the content of this the main readme. we can do that later when we have refactored the dir structure to flatten all the connector dirs rather than keep everything under `connectors/`